Sources/SwiftParserDiagnostics/ParseDiagnosticsGenerator.swift

```diff
@@ -645,13 +645,52 @@ public class ParseDiagnosticsGenerator: SyntaxAnyVisitor {
     }
 
     if node.label.isMissing {
+      let replaceTokens: [TokenSyntax] = [node.unexpectedBetweenLabelAndColon].compactMap { $0?.tokens(viewMode: .sourceAccurate) }.flatMap { $0 }
+
+      let versionMessage: ParserFixIt =
+        node.unexpectedBetweenLabelAndColon == nil
+        ? InsertFixIt(tokenToBeInserted: .identifier("_version"))
+        : ReplaceTokensFixIt(replaceTokens: replaceTokens, replacements: [.identifier("_version"), .colonToken()])
+
+      let underlyingVersionMessage: ParserFixIt =
+        node.unexpectedBetweenLabelAndColon == nil
+        ? InsertFixIt(tokenToBeInserted: .identifier("_underlyingVersion"))
+        : ReplaceTokensFixIt(replaceTokens: replaceTokens, replacements: [.identifier("_underlyingVersion"), .colonToken()])
+
+      let fixIts = [
+        FixIt(
+          message: versionMessage,
+          changes: [
+            FixIt.MultiNodeChange(
+              .replace(
+                oldNode: Syntax(node.label),
+                newNode: Syntax(TokenSyntax(.identifier("_version"), presence: .present))
```

Comment:
One more request: Can you run `MissingNodesBasicFormatter` on this node? That should fix the test case below where the fixed source is `A,_version` and doesn’t contain a space between `,` and `_version`. Same below for `_underlyingVersion`.

And while you’re at it, you could also add the following initializer to `MissingNodesBasicFormatter` because it never makes sense to run it on a source accurate tree.
```swift
init() {
  super.init(viewMode: .fixedUp)
}
```

Reply:
What do you think about adding a `makeReplaceWithTrivia` function (as my new commit) to do this?